src/sb_gazebo/robots/rover_2018.xacro

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0"?>
+<!--
+     The model for the 2018 rover
+-->
+<robot name="rover_2018" xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+    <!-- CALCULATIONS
+         This section holds all the math to calculate the offset (both linear
+         and angular) for various components. 
+    -->
+    <xacro:include filename="$(find sb_gazebo)/robots/macros.xacro"/>
+    <xacro:include filename="$(find sb_gazebo)/robots/materials.xacro"/>
+    <xacro:include filename="$(find sb_gazebo)/robots/rover_2018.gazebo"/>
+
+    <xacro:property name="chassis_x_size" value="0.7"/>
+    <xacro:property name="chassis_y_size" value="0.94"/>
+    <xacro:property name="chassis_z_size" value="0.4"/>
+    <xacro:property name="chassis_mass" value="10"/>
+    <xacro:property name="chassis_elevation" value="0.5"/>
+    <xacro:property name="chassis_x_pos" value="0"/>
+    <xacro:property name="chassis_y_pos" value="0"/>
+    <xacro:property name="chassis_z_pos" value="${chassis_z_size + chassis_elevation}"/>
+
+    <xacro:property name="primary_rocker_radius" value="0.15"/>
+    <xacro:property name="primary_rocker_length" value="0.1"/>
+    <xacro:property name="primary_rocker_chassis_clearance" value="0.02"/>
+    <xacro:property name="primary_rocker_x_offset" value="0"/>
```

Comment:
What are these offsets relative to?

Reply:
Hmm. So they're from the chassis. Not sure on the best way to doc. this, but added a comment above:
```
    <!-- Primary rocker offset from it's parent joint (the chassis) -->
```